### PR TITLE
life inprovements: need to double the bing chilling effect

### PR DESCRIPTION
**Pull request by Du-ma**
Sussy bakiiii

### CODE REVIEW

**Comment by Du-ma** [comment]
asd

**Comment by Du-ma** [comment]
ad

**Comment by Du-ma** [comment]
asd

**Comment by Du-ma** [comment]
asd

**Comment by Du-ma** [comment]
[SC](https://soundcloud.com)
**BIM**
# BAM
`kodelis`
```python
print("kodukas")
```

**Comment by Du-ma** [comment]
helou?

**Comment by Du-ma** [comment]
[SC](https://soundcloud.com)
**BIM**
# BAM
`kodelis`
```python
print("kodukas")
```

**Comment by Du-ma** [comment]
[SC](https://soundcloud.com)
**BIM**
# BAM
`kodelis`
```python
print("kodukas")
```

**Comment by gaikaz** [comment]
Peek 2022-03-25 18-23

**Comment by Du-ma** [comment]
Peek 2022-03-25 18-23

**Comment by gaikaz** [comment]
https://user-images.githubusercontent.com/17097810/160161291-53d33930-4b86-4fa8-ba4b-3590b6adf334.mp4

**Comment by gaikaz** [comment]
321?

**Comment by gaikaz** [comment]
https://user-images.githubusercontent.com/17097810/160161466-e588e9d2-566c-4f16-9dbc-090444fe3b05.mp4

**Comment by Du-ma** [comment]
ayy

**Review by Mantux11 — APPROVED** [review]

File: bing_chilling_file.py [hunk 1]
```diff
@@ -1 +1,3 @@
-print("BING CHILLING")
\ No newline at end of file
+print("BING CHILLING")
```
Comment (husq570) [inline bing_chilling_file.py:1]: asdasd

**Review by husq570 — CHANGES_REQUESTED** [review]

File: bing_chilling_file.py [hunk 2]
```diff
@@ -1 +1,3 @@
...
+
+print("BING CHILLING")
```
Comment (gaikaz) [inline bing_chilling_file.py:3]:
```suggestion
print("BING NETFLIX AND CHILLING")
```

**Review by Mantux11 — CHANGES_REQUESTED** [review]
Pimpampufpuf

**Comment by Mantux11** [comment]
dssddsd

**Comment by Mantux11** [comment]
fsdfdsfsd

**Review by Mantux11 — COMMENTED** [review]
fsddsfsd

**Comment by Du-ma** [comment]
asd

**Comment by Du-ma** [comment]
oiu

**Comment by Du-ma** [comment]
asd

**Comment by Du-ma** [comment]
adadssad

**Comment by Du-ma** [comment]
asd

**Comment by Du-ma** [comment]
dsdsdsdsds

**Comment by husq570** [comment]
whatcha-doin

**Comment by Du-ma** [comment]
asd

**Comment by Du-ma** [comment]
asd

**Comment by Du-ma** [comment]
:)

**Comment by Du-ma** [comment]
:penguin:

**Comment by Du-ma** [comment]
Oupen sezamy

**Pull request merged**